Source/Shift/Classes/SplitTransition.swift

```diff
@@ -88,19 +102,132 @@ public class SplitTransition: NSObject {
     }()
 
     /**
+     * In an interactive transition, splitOffset
+     * determines the vertical distance of the initial
+     * split when the cell is first tapped
+     */
+    public var splitOffset: CGFloat = 0.0
+
+    /**
```

Comment:
Now I'm just getting nerdy, so feel free to ignore me. Single line comments can use the `///` syntax, and multi-line comments look like this now:
```swift
/**
This is a super cool thing.

- Important: do not taunt the happy fun ball.
*/
```